README.md

```diff
@@ -1,2 +1,22 @@
 # FinalReport_filter
 Software to filter Illumina Genomestudio FinalReport files
+
+### Make virtual python enviroment
+```bash
+python3 -m venv venv
+source venv/bin/activate
+pip install --upgrade pip
+pip install -r requirements.txt
+deactivate
+```
+
+### Make Windows executable 
+Run pyinstaller to convert .py to Windows executable (.exe).\
+Note: this must be done a Windows machine (i.e. the SNP-array_data_cruncher Windows)\
+Pyinstaller on SNP-array_data_cruncher has been tested using python v3.7.9 (32bit)
+```bash
+python
```

Comment:
Is this the correct way in Windows? do you open the cmd tool and just put "python" there?
could be that this is true, but I have the feeling a lot of steps are missing here

Reply:
Removed Windows stuff from README.